### PR TITLE
Zephyr blog 

### PR DESCRIPTION
**Pull request by Myralllka**
- typo fixes
- grammar fixes
- remove deprecated info
- add text formatting
- add code formatting

### CODE REVIEW

**Review by tkurnyt — APPROVED** [review]

File: Zephyr_RTOS/Zephyr_intro.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
 	
 4.  Communication interfaces and device drivers
 	Zephyr support presently includes Bluetooth, Bluetooth LE 5.0, Ethernet, 802.15.4, Wi-Fi, IPv4/IPv6, 6LoWPAN, Thread, and NFC. 
 	
 5. Sensor support
-	Expectedly, Zephyr has great sensors support with high level of abstraction. The process goes smoothly during enabling sensor which is already supported by Zephyr. List of supported boards and sensors is very rich. However,  if new sensor is to be enabled on a custom board the process turns out to be a little complicated. An issue of implementing new sensor support on Zephyr will be addressed in detail in the next blog.
+	Expectedly, Zephyr has great sensors support with a high level of abstraction. The process goes smoothly during enabling sensor which is already supported by Zephyr. List of supported boards and sensors is very rich. However,  if a new sensor is to be enabled on a custom board, the process turns out to be a little complicated. An issue of implementing new sensor support on Zephyr will be addressed in detail in the next blog.
 	
 6. Zephyr support and updates.
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
6. **Zephyr support and updates.**
```

File: Zephyr_RTOS/Zephyr_intro.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
+	Being intrinsically cross-platform project, Zephyr naturally uses the CMake build system. It also extensively uses a command-line tool named ***west*** [[Ref]](https://docs.zephyrproject.org/latest/guides/west/index.html). West is used for building, flashing and debugging applications as well as Zephyr repository manager.
+	In general, application configuration step is rather complicated in Zephyr. It extensively uses multi-level hierarchical configuration approach. [Here](https://docs.zephyrproject.org/latest/guides/build/index.html) is the link to the configuration build overview from the official Zephyr documentation page. It will be considered in more details in the next blog. For now, it’s worth stating that it includes Device Tree Compilation step resulting in the generation of application-specific header files – user need not writing header file manually.
 	
 4.  Communication interfaces and device drivers
 	Zephyr support presently includes Bluetooth, Bluetooth LE 5.0, Ethernet, 802.15.4, Wi-Fi, IPv4/IPv6, 6LoWPAN, Thread, and NFC. 
 	
 5. Sensor support
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
5. **Sensor support**
```

File: Zephyr_RTOS/Zephyr_intro.md [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
+	Being intrinsically cross-platform project, Zephyr naturally uses the CMake build system. It also extensively uses a command-line tool named ***west*** [[Ref]](https://docs.zephyrproject.org/latest/guides/west/index.html). West is used for building, flashing and debugging applications as well as Zephyr repository manager.
+	In general, application configuration step is rather complicated in Zephyr. It extensively uses multi-level hierarchical configuration approach. [Here](https://docs.zephyrproject.org/latest/guides/build/index.html) is the link to the configuration build overview from the official Zephyr documentation page. It will be considered in more details in the next blog. For now, it’s worth stating that it includes Device Tree Compilation step resulting in the generation of application-specific header files – user need not writing header file manually.
 	
 4.  Communication interfaces and device drivers
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
4.  **Communication interfaces and device drivers**
```

File: Zephyr_RTOS/Zephyr_intro.md [hunk 4]
```diff
@@ -1,49 +1,48 @@
...
 	
 2. Scalability
-	Zephyr is a highly configurable and modular OS that implements memory protection (even for platforms without an MMU/MPU). It uses Device Tree Support (DTS) only during compile time. Powerful configuration tool allows flexible inclusion of only those features which are really needed in specific application. Memory footprint can be as low as 8k.
+	Zephyr is a highly configurable and modular OS that implements memory protection (even for platforms without an MMU/MPU). It uses Device Tree Support (DTS) only during compile time. Powerful configuration tool allows flexible inclusion of only those features which are needed in a specific application. Memory footprint can be as low as 8k.
 	The other specific feature of Zephyr is that it has only one address space. It means that the application code and the kernel are combined in the same binary compilation. 
 	
 3. Build toolchain
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
3. **Build toolchain**
```

File: Zephyr_RTOS/Zephyr_intro.md [hunk 5]
```diff
@@ -1,49 +1,48 @@
...
 
-Generally, choice of an embedded OS heavily depends on the specific use case. Therefore, there is no single recipe or single OS of preference to be recommended. Discussion of pros and cons of different OS is out of scope of this blog and focus is made on Zephyr OS specifics and author’s own experience acquired while working with Zephyr OS.
-Like many other operating systems, Zephyr provides:
-
-Still, what makes Zephyr OS unique as compared to other OS? Below is the brief list of features.
+Generally, the choice of an embedded OS heavily depends on the specific use case. Therefore, there is no single recipe or a single OS to be recommended. Discussion of the pros and cons of a different OS’s is out of the scope of this blog and focus is made on Zephyr specifics, and the author’s own experience acquired while working with this system.
 
+### What makes Zephyr OS unique as compared to other OS?
 
 1. Supported boards
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
1. **Supported boards**
```

File: Zephyr_RTOS/Zephyr_intro.md [hunk 6]
```diff
@@ -1,49 +1,48 @@
...
+	At the moment this blog is written several boards supported by Zephyr were around 170, much more than any another embedded OS has. Why is it important? Because it is very convenient. If due to some reasons decision is made to move to another hardware device the one thing to do (at least theoretically) is to inform Zephyr compiler for which board the code has to be compiled. Otherwise, without embedded OS supporting multiple boards, it is just the start of a painful process of getting familiar with board specifics, OS specifics, building toolchain, IDE etc. Thus, Zephyr essentially reduces time and efforts while moving to another board.
 	
 2. Scalability
```
Comment (abench-softserve) [inline Zephyr_RTOS/Zephyr_intro.md]:
```suggestion
2. **Scalability**
```

File: Zephyr_RTOS/Zephyr_intro.md (hunk 1 above)
Comment (abench-softserve) [reply Zephyr_RTOS/Zephyr_intro.md]: Use Bold for unique item names

**Review by abench-softserve — CHANGES_REQUESTED** [review]

**Comment by abench-softserve** [comment]
Changes were approved by author. I am merging this pull request into master.

**Pull request merged**